src/jade/popup.jade

```diff
@@ -1,26 +1,26 @@
 doctype html
-html(class="dollert-popup")
+html.cp-popup
   head
     title Dollert
     link(rel="stylesheet" type="text/css" href="popup.min.css")
 
   body
-    .dollert-alert-form
-      .dollert-logotype-container
+    .cp-alert-form
+      .cp-logotype-container
         img(src="logotype.png")
-      .dollert-alert-form-label Enter the USD value you are looking for in BRL:
-      .dollert-input-buttoned
+      .cp-alert-form-label Enter the USD value you are looking for in BRL:
+      .cp-input-buttoned
         input(data-js="usd-value" placeholder="i.e: 3.70")
         button.is-primary(data-js="button-save" type="button") Save
-    .dollert-now-value-container
-      span.dollar-now-value-label Now
-      span.dollar-now-value(data-js="now-value") Loading value...
-      span.dollar-now-variation(data-js="now-variation")
+    .cp-current-usd-value-container
+      span.cp-current-usd-value-label Now
```

Comment:
What do you think about removing the prefix usd from those classes? Leaving them just `cp-current-value-label` for example. The class name still shows very clearly what this component does, and if we ever work with other currencies besides dollar it will work out of the box.